AsyncDisplayKit/Details/ASWeakSet.m

```diff
@@ -22,7 +22,7 @@ - (instancetype)init
 {
   self = [super init];
   if (self) {
-    _hashTable = [NSHashTable weakObjectsHashTable];
```

Comment:
So did this previously de-duplicate objects by using isEqual?  slow and shitty!  lol